### ISSUE TITLE
Format checkbox lists or allow multi selection

### ISSUE THREAD

**Issue by yosnap** [Crocoblock/suggestions#1370]
When setting checkboxes in forms for choosing multiple options from terms, they use a big part on form if there are many categories. Please give us a way to customize it or choosing another selector like CPT (make a text field, write and autocompleting if the term already exists)

![Captura de pantalla 2020-06-01 a las 13 43 06](https://user-images.githubusercontent.com/66257788/83443541-34e24280-a44a-11ea-8021-dc5942580ede.png)
![Captura de pantalla 2020-06-01 a las 20 57 46](https://user-images.githubusercontent.com/66257788/83443779-986c7000-a44a-11ea-97c0-7c7363d8f683.png)
The second screenshot is better because I can select more than 1 option

Please improve it in future update

**Comment by dionisis17** [comment]
I agree. I have the same problem. [This feature](https://facetwp.com/documentation/facets/facet-types/fselect/) I believe will solve this if they add it

**Comment by stasonua0** [comment]
+1

**Comment by aalexandr21** [comment]
The Dynamic Generator was updated in JetFormBuilder 3.6.0 
Check the video on how to do it -> https://youtu.be/ZB94fY2aG4I
